### ISSUE TITLE
Community-Maintained Version of `mordred`

### ISSUE THREAD

**Issue by JacksonBurns** [mordred-descriptor/mordred#109]
`mordred` is important to many cheminformatics tasks but is no longer being maintained. I have made a fork of `mordred` called [`mordred-community`](https://www.github.com/JacksonBurns/mordred-community) and am happy to review and merge PRs!

So far I have taken all of the outstanding changes on this repo and relased mordredcommunity on PyPI: `pip install mordredcommunity`

**Comment by TUstudents** [comment]
"That's a fantastic idea! I'm glad you're taking the lead on this.

**Comment by TUstudents** [comment]
Will you add the issues tracker and discussions page on your fork?

**Comment by JacksonBurns** [comment]
Just did, thanks for the suggestion @TUstudents

**Comment by TUstudents** [comment]
Thanks for fixing the issue with recent numpy versions, so I do not even need to write a bug report :)

As you created the [pypi ](https://pypi.org/project/mordredcommunity/) package, basically the road is open to have a conda version (custom channel) too?

**Comment by JacksonBurns** [comment]
Yes, I will get it on conda shortly. I'd like to get it on conda-forge as well.

**Comment by JacksonBurns** [comment]
If anyone sees this in the future, `mordredcommunity` is now available via `conda-forge`.
